pytest fails also because RTTOV wrapper is not found

```
platform linux -- Python 3.10.10, pytest-8.1.1, pluggy-1.4.0
rootdir: /scratch/b/b380352/SynSatiPy
collected 12 items                                                                                                                                             

tests/test_data_handler.py ...                                                                                                                           [ 25%]
tests/test_example_data.py .                                                                                                                             [ 33%]
tests/test_instruments.py FFFFFF                                                                                                                         [ 83%]
tests/test_realistic_synsat.py F                                                                                                                         [ 91%]
tests/test_synsat.py F                                                                                                                                   [100%]

...
        attr.atlasses_loaded = False
        attr.kwargs = synsat_kwargs
    
        # locate itself
>       pyrttov_path = pyrttov.__path__[0]
E       AttributeError: 'pyrttov_Empty' object has no attribute '__path__'. Did you mean: '__hash__'?

...
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,10 +1,14 @@
-# Python dependencies
-numpy
-scipy
-pandas
-h5py
-netcdf4
-xarray
-dask
-cartopy
-matplotlib
+numpy<2
+bokeh==3.4.0
+dask-expr==1.0.9
+distributed==2024.4.0
+easygems==0.0.7
+GitPython==3.1.43
+goes2go==2025.2.0
+intake-esm==2024.2.6
+intake-xarray==0.7.0
+ipykernel==6.29.3
+lz4==4.3.3
+pyarrow-hotfix==0.6
+pydocstyle==6.3.0
+pytest==8.1.1

